packages/block-library/src/latest-posts/index.php

```diff
@@ -13,39 +13,39 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
+
 	$args = array(
-		'numberposts' => $attributes['postsToShow'],
-		'post_status' => 'publish',
-		'order'       => $attributes['order'],
-		'orderby'     => $attributes['orderBy'],
+		'posts_per_page'   => $attributes['postsToShow'],
+		'post_status'      => 'publish',
+		'order'            => $attributes['order'],
+		'orderby'          => $attributes['orderBy'],
+		'suppress_filters' => false,
 	);
 
 	if ( isset( $attributes['categories'] ) ) {
 		$args['categories'] = $attributes['categories'];
```

Comment:
`get_posts` offers a different interface, as it expects `category` — singular — rather than `categories`.

```suggestion
		$args['category'] = $attributes['categories'];
```

Reply:
@mcsf Thanks for notice this, added a fix